src/search/indexing/local_import.rs

```diff
@@ -0,0 +1,54 @@
+use crate::search::indexing::IndexingError;
+use meilisearch_sdk::client::Client;
+use crate::search::models::{SearchRequest, SearchMod};
+use bson::Bson;
+use futures::StreamExt;
+use crate::database::models::Item;
+use crate::database::{Version, Mod};
+
+pub async fn index_local(client: mongodb::Client) -> Result<Vec<SearchMod>,  IndexingError> {
+    info!("Indexing local mods!");
+
+    let mut docs_to_add: Vec<SearchMod> = vec![];
+
+    let db = client.database("fabricate");
```

Comment:
Database should be renamed to match name change

Reply:
I'm modifying it